vectorx/font/TextLine.hx

```diff
@@ -282,7 +290,7 @@ class TextLine
                 lastSpanInLine.updateString();
             }
 
-            if(lastSpanInLine.haveShadow() && lastSpanInLine.attachmentId == null)
+            if(lastSpanInLine.haveShadow() /* && lastSpanInLine.attachmentId == null*/)
```

Comment:
Now calculation of line width would be broken in case there is an attachment at the end of a line.